rewards/divvi-integrators/1745107200000-1747181107000/safe-transactions.json

```diff
@@ -0,0 +1,36 @@
+{
+  "meta": {},
+  "transactions": [
+    {
+      "to": "0x326161d68c05bE55367a0041b9C8f68082C04863",
```

Comment:
Is this the right address?
CeloScan says it's an EOA, is it an indexing issue?
https://celoscan.io/address/0x326161d68c05bE55367a0041b9C8f68082C04863

Reply:
ah yes it's optimism 🙈